vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/ItemClickEvent.java

```diff
@@ -101,4 +107,15 @@ public T getItem() {
         return item;
     }
 
+    /**
+     * Gets the column that was clicked.
+     * For this to really work the column's id must be set. Otherwise internal magic will be used
+     * to figure out which column it is (using column's internal flow identifiers that are then
+     * converted to integer index of the column).
+     * @return An optional with column. Optional may be empty if the column could not be figured out.
```

Comment:
JavaDocs should be updated.

It should say that the optional is empty when hitting a details-cell.

Reply:
Done.